types-tests/options.test-d.ts

```diff
@@ -50,7 +50,7 @@ expectType<void>(
   app.options<'interactive_message'>({ callback_id: 'a' }, async ({ options, ack }) => {
     expectType<InteractiveMessageSuggestion>(options);
     // https://github.com/slackapi/bolt-js/issues/720
-    expectError(ack({ options: blockSuggestionOptions }));
+    ack({ options: blockSuggestionOptions });
```

Comment:
This test change should be valid as the app.options pattern here now works properly